x-pack/plugins/security_solution/server/endpoint/lib/artifacts/saved_object_mappings.ts

```diff
@@ -34,3 +34,31 @@ export const manifestType: SavedObjectsType = {
   mappings: manifestSavedObjectMappings,
   migrations,
 };
+
+export const unifiedManifestSavedObjectType = ManifestConstants.UNIFIED_SAVED_OBJECT_TYPE;
+
+export const unifiedManifestSavedObjectMappings: SavedObjectsType['mappings'] = {
+  dynamic: false,
+  properties: {
+    artifactIds: {
+      type: 'text',
+    },
+    policyId: {
+      type: 'keyword',
+    },
+    semanticVersion: {
+      type: 'text',
+    },
+    created: {
+      type: 'date',
+    },
+  },
+};
+
+export const unifiedManifestType: SavedObjectsType = {
+  name: unifiedManifestSavedObjectType,
+  indexPattern: SECURITY_SOLUTION_SAVED_OBJECT_INDEX,
+  hidden: false,
+  namespaceType: 'multiple',
```

Comment:
Went with `multiple` due to our plans to introduce space awareness. Following the docs:
```The namespace type dictates how a saved object can be interacted in relation to namespaces. Each type is mutually exclusive:
 * single (default): This type of saved object is namespace-isolated, e.g., it exists in only one namespace.
 * multiple: This type of saved object is shareable, e.g., it can exist in one or more namespaces.
 * multiple-isolated: This type of saved object is namespace-isolated, e.g., it exists in only one namespace, but object IDs must be
   unique across all namespaces. This is intended to be an intermediate step when objects with a "single" namespace type are being
   converted to a "multiple" namespace type. In other words, objects with a "multiple-isolated" namespace type will be *share-capable*,
   but will not actually be shareable until the namespace type is changed to "multiple".
 * agnostic: This type of saved object is global.```
 I believe the `multiple` is the most fitting option, future wise.

Reply:
great thinking! 🎉